x/oracle/types/tally.go

```diff
@@ -0,0 +1,173 @@
+package types
+
+import (
+	"container/heap"
+	"fmt"
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+type (
+	Vote interface {
```

Comment:
You're right. I will fix soon.

Reply:
To be honest, I don't prefer this C++ style interface naming: `FooI`. Many Go libraries (including Go official libs) don't use that naming, but if you think `FooI` is better to avoid confusions in our codebase, I wouldn't argue with it.

Reply:
If code structure is well-organized, users of our codes shouldn't need to worry whether "something" is an interface or a struct. That's the art of abstraction.

Reply:
Ok. Then, let's leave as is